less/bootstrap-override/variables.less

```diff
@@ -93,6 +93,17 @@
 @btn-default-bg-disabled-end:    #DAE5ED;
 @btn-default-border:             #B5C6D2;
 
+@btn-basic-color:              #656565;
+@btn-basic-bg-start:           #fefefe;
+@btn-basic-bg-end:             #e0e0e0;
+@btn-basic-bg-hover-start:     #F4F1F1;
+@btn-basic-bg-hover-end:       #CBCACA;
+@btn-basic-bg-press-start:     #E0E0E0;
+@btn-basic-bg-press-end:       #EBEBEB;
+@btn-basic-bg-disabled-start:  #FFF;
+@btn-basic-bg-disabled-end:    #D7D7D7;
+@btn-basic-border:             #bbb;
```

Comment:
revisit variables to see if all these are needed and if there are any color pallet variables that could be mapped to. #299